pyscript.core/docs/README.md

```diff
@@ -229,11 +229,11 @@ Please read the [Terminology](#terminology) **target** dedicated details to know
   <summary><strong>XWorker</strong></summary>
   <div>
 
-With or without access to the `document`, every (*non experimental*) interpreter will have defined, at the global level, a reference to the `XWorker` "_class_" (it's just a *function*!), which goal is to enable off-loading heavy operations on a worker, without blocking the main / UI thread (the current page) and allowing such worker to even reach the `document` or anything else available on the very same main / UI thread.
+With or without access to the `document`, every (*non experimental*) interpreter will have defined, either at the global level or after a `from xworker import XWorker`, a reference to the `XWorker` "_class_" (it's just a *function*!), which goal is to enable off-loading heavy operations on a worker, without blocking the main / UI thread (the current page) and allowing such worker to even reach the `document` or anything else available on the very same main / UI thread.
 
 ```html
 <script type="micropython">
-    # XWorker is globally defined
+    from xworker import XWorker
```

Comment:
As discussed today, this is a totally fine module name for now even if it may end up with another name or also available in the `pyscript` module itself (just want to repeat it out lout to make sure we are on the same page :) )

Reply:
I think `pyscript` module has a good place in ... 🥁🥁🥁 ... **pyscript** custom type :D ... this is core though, and we discussed this was a good name indeed for the core offer.

Reply:
if this wasn't clear, **yes**, we can offer the same from **pyscript** custom type so there's no conflict of interest in here, I suppose